TEAMS.md

```diff
@@ -24,6 +24,7 @@ Please [reach out in any of the available ways](https://github.com/open-gitops/.
 | content/gitops-environmental-sustainability team | active | [current discussion](https://github.com/open-gitops/project/issues/129) | <ul><li>[@JaimeMagiera](https://github.com/JaimeMagiera)</li><li>[@nikimanoledaki](https://github.com/nikimanoledaki)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@nishadmehendale](https://github.com/nishadmehendale)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@Al-HusseinHameedJasim](https://github.com/Al-HusseinHameedJasim)</li></ul> |
 | marketing/media | active | [current discussion](https://github.com/open-gitops/project/discussions/22) | <ul><li>[@christianh814](https://github.com/christianh814)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@staceypotter](https://github.com/staceypotter)</li><li>[@todaywasawesome](https://github.com/todaywasawesome)</li></ul> |
 | marketing/events | active | [purpose](https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#team-members) (also see [#151](https://github.com/open-gitops/project/issues/151)) | <ul><li>[@christianh814](https://github.com/christianh814)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@roberthstrand](https://github.com/roberthstrand)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@staceypotter](https://github.com/staceypotter)</li></ul> |
+| website | active | [purpose](https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#team-members) (also see [#151](https://github.com/open-gitops/project/issues/155)) | <ul><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@JaimeMagiera](https://github.com/JaimeMagiera)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@varodrig](https://github.com/varodrig)</li></ul> |
```

Comment:
requesting two changes:
- no need to copy the previous info column format (tbh can't remember why we did that)
- please remove Jaime (he said he [can't commit](https://cloud-native.slack.com/archives/C026D3RE7FB/p1722890051417599?thread_ts=1722887707.550829&cid=C026D3RE7FB) right now)

```suggestion
| website | active | [current discussion](https://github.com/open-gitops/project/issues/155) | <ul><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@varodrig](https://github.com/varodrig)</li></ul> |
```

Reply:
done! thank you for reviewing